### PR TITLE
Improvement/bombinfo

### PR DESCRIPTION
**Pull request by kramar42**
addresses #16

### CODE REVIEW

File: modules/bomb_word.py [hunk 1]
```diff
@@ -63,7 +64,8 @@ def bomb_info_payload_generator(chat_data):
             + '\n   жертв: %d' % casualties_count
             + '\nосталось: %s' % expiration_readable
             + '\n  сосеры: %s'
-            % '\n          '.join(casualties_list_str))
+            % '\n          '.join(casualties_list_str) if casualties_list_str
```
Comment (rbietin) [inline modules/bomb_word.py]: I'd prefer parentheses to make `'\n  сосеры: %s' % '\n          '.join(casualties_list_str)` easier to read, but up to you

File: modules/chat.py [hunk 2]
```diff
@@ -19,11 +19,11 @@ def chat(bot, update, chat_data, job_queue):
...
         bomb_word = get_module('bomb_word')
         text = bomb_word.normalize_text(update.message.text)
+        bombers = set()
         for bomber, bombinfo in chat_data['bombs'].items():
             log.debug('word: ' + bombinfo['word'])
-            if bombinfo['word'] in text:
-                bomb_word.bomb_triggered(bot, job_queue, update, chat_data, bomber)
-                break
+            if bombinfo['word'] in text: bombers.add(bomber)
+        bombers and bomb_word.trigger_bombers(bot, job_queue, update, chat_data, bombers)
```
Comment (rbietin) [inline modules/chat.py]: wouldn't the explicit check be more readable? you literally win nothing out of short-circuiting.

Comment (rbietin) [reply modules/chat.py]: not a deal breaker though, just looks strange and unnecessary.

File: modules/bomb_word.py (hunk 1 above)
Comment (kramar42) [reply modules/bomb_word.py]: sure, something like
```python
+ '\nосталось: %s' % expiration_readable
+('\n  сосеры: %s'
% '\n          '.join(casualties_list_str) if casualties_list_str
                                           else ''))
```
?

File: modules/chat.py (hunk 2 above)
Comment (kramar42) [reply modules/chat.py]: it's like writing ```if my_set:``` to check if set is not empty. idiomatic for me. others' thoughts?

Comment (1over) [reply modules/chat.py]: can't we just pass any set to trigger_bombers and inside check if it's empty or not and process accordingly?

**Comment by rbietin** [comment]
are you going to merge this one?

**Pull request merged**